spec/models/review_spec.rb

```diff
@@ -0,0 +1,6 @@
+require 'rails_helper'
+
+describe Review do
+  it { should belong_to(:user) }
+  it { should belong_to(:food_truck) }
```

Comment:
Can you add some model tests here against your validations? i.e. Make sure the rating is valid when it is an integer between 1 and 5, make sure it is invalid otherwise?